backend/src/kitconcept/intranet/upgrades/v20250909001/configure.zcml

```diff
@@ -0,0 +1,19 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:genericsetup="http://namespaces.zope.org/genericsetup"
+    >
+  <genericsetup:upgradeSteps
+      profile="kitconcept.intranet:default"
+      source="20250903002"
+      destination="20250909001"
+      >
+    <genericsetup:upgradeDepends
+        title="Upgrade collective.person"
+        import_profile="collective.person:default"
```

Comment:
This imports the entire profile, rather than running its upgrade steps. That doesn't seem very safe to me...it does things like resetting the Roles vocabulary and rolemap.